### PR TITLE
Fix python lxml

### PR DESCRIPTION
**Pull request by alijawadfahs**
A fix for the python-lxml package issue caused by the drop of support in ubuntu 20

### CODE REVIEW

File: palinagent/daemon/main.py [hunk 1]
```diff
@@ -47,8 +47,28 @@
...
+                zip_ref.extractall(packages_path)
+        else:
+            with zipfile.ZipFile(packages_path+"/lxml-4.6.3-cp27-cp27mu-manylinux1_i686.whl","r") as zip_ref:
+                zip_ref.extractall(packages_path)
+        sys.path.append(packages_path)
+        from lxml import etree
+    except (ImportError), e:
+        #importing using the wheel packages failed
+        print >> sys.stderr, AgentSetupError("lxml not available: %s" % e)
+        sys.exit(7)
```
Comment (nebilbenmabrouk) [inline palinagent/daemon/main.py:71]: Why using the exit code 7 :)

File: palinagent/daemon/main.py [hunk 2]
```diff
@@ -47,8 +47,28 @@
 try:
     from lxml import etree
 except (ImportError), e:
-    print >> sys.stderr, AgentSetupError("lxml not available: %s" % e)
-    sys.exit(7)
+    # lxml is not installed
```
Comment (nebilbenmabrouk) [inline palinagent/daemon/main.py:50]: If possible, add the information about the architecture and the installed lxml package in the method **_log_debug_info(logger)**

**Review by fviale — APPROVED** [review]

**Review by mboussaa — APPROVED** [review]

File: palinagent/daemon/main.py (hunk 1 above)
Comment (alijawadfahs) [reply palinagent/daemon/main.py:71]: this part was left as original normally I would use exit code 1 but I thought there is a purpose of exit code 7

**Review by nebilbenmabrouk — APPROVED** [review]

File: palinagent/daemon/main.py (hunk 2 above)
Comment (alijawadfahs) [reply palinagent/daemon/main.py:51]: added them

**Pull request merged**